City-of-Helsinki/drupal-helfi-kymp
# UHF-1334 Drupal configuration

How to test:
- `git checkout UHF-1334_drupal_configuration`
- `make new`
- Check that the site works and check that the site name has been updated https://helfi-kymp.docker.sh/en/admin/config/system/site-information
